test/zod/test_zod.ts

```
@@ -0,0 +1,40 @@
+    },
+    lang: "en",
+    beats: [
+      {
+        speaker: "Presenter",
+        text: "",
+        imagePrompt: "",
+      },
+    ],
+  };
+  const data = mulmoScriptSchema.safeParse(initMulmoScript);
+  console.log(JSON.stringify(data));
```

ESLint

[error] Unexpected console statement.

(no-console)